tests/integrationv2/test_happy_path.py

```diff
@@ -0,0 +1,90 @@
+import copy
+import os
+import pytest
+import time
+
+from configuration import available_ports, CIPHERSUITES, CURVES
+from common import ProviderOptions
+from fixtures import managed_process
+from providers import S2N, provider_list
+
+
+@pytest.mark.parametrize("cipher", CIPHERSUITES)
+@pytest.mark.parametrize("curve", CURVES)
+@pytest.mark.parametrize("provider", provider_list)
+def test_s2n_server_happy_path(managed_process, cipher, curve, provider):
```

Comment:
Can you send some data over the connection, to ensure we've actually connected? We've had bugs before bc the happy case didn't do that.

Reply:
Yup. All that is planned.